### PR TITLE
perf: website enhancement

### PR DESCRIPTION
**Pull request by serena-ruan**

### CODE REVIEW

**Comment by serena-ruan** [comment]
Waiting for merging this PR: https://github.com/microsoft/SynapseML/pull/1241  -- then finish the change.

**Comment by mhamilton723** [comment]
-5k lines holy hell i love it

**Comment by serena-ruan** [comment]
\azp run

File: core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala [hunk 1]
```diff
@@ -59,9 +59,22 @@ object DatabricksUtilities extends HasHttpClient {
   // Execution Params
   val TimeoutInMillis: Int = 40 * 60 * 1000
 
-  val NotebookFiles: Array[File] = Option(
-    FileUtilities.join(BuildInfo.baseDirectory.getParent, "notebooks").getCanonicalFile.listFiles()
-  ).get
+  val NotebookFiles: Array[File] = {
```
Comment (mhamilton723) [inline core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala:62]: nit: could remove mutability with recursion

https://stackoverflow.com/questions/2637643/how-do-i-list-all-files-in-a-subdirectory-in-scala

**Comment by mhamilton723** [comment]
@serena-ruan i think you removal of all the files was reset

File: core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala (hunk 1 above)
Comment (serena-ruan) [reply core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala:62]: Thanks! That's what I wanted >_<

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**